components/butterfly/src/member.rs

```diff
@@ -1104,8 +1066,9 @@ mod tests {
                     "Update counter should have incremented by one"
                 );
 
-                assert!(
-                    ml.check_health_of(&member, from_health.clone()),
+                assert_eq!(
+                    ml.health_of(&member),
+                    Some(from_health.clone()),
```

Comment:
I'd be in favor. Want me to add it to this PR or do it in a separate one?

Reply:
It looks like we *do* derive `Copy` for health:
```rust
    #[derive(Clone, Copy, Debug, PartialEq, Eq, Hash, PartialOrd, Ord, Enumeration)]
    #[derive(Serialize, Deserialize)]
    pub enum Health {
```
—[butterfly.swim.rs#L53-L55](https://github.com/habitat-sh/habitat/blob/master/components/butterfly/src/generated/butterfly.swim.rs#L53-L55)

I'll remove the unnecessary calls to `clone()`, though. I might as well, since AppVeyor just keeps timing out anyway.